department-of-veterans-affairs/va.gov-cms
# VACMS-3852: QA Form Save

…play no default paragraph type, added logic to make the uri field within the field_related_information field as optional so that a valid error appears if and editor decided to switch between a standalone QA and had added a related_information link

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3852

## Testing done


## Screenshots


## QA steps

- Login to the tugboat preview environment
- Navigate here to add a new QA node: `node/add/q_a`
- Complete the Question,Answer and Owner fields
- [x] Select SAVE and confirm you are able to save with no issues

- Select EDIT
- Select the `Enable standalone Resources and support page for this Q&A.` button
- [x] Navigate to the 'Related Information' section and confirm that a default link teaser paragraph is not visible.
- Complete all field
- [x] Select SAVE and confirm you are able to save with no issues

- Select EDIT
- Navigate to the 'Related Information' section and select the `Add Link teaser` button
- Complete `Related Information` fields
- [x] Select SAVE and confirm you are able to save with no issues

- Select EDIT
- Navigate to the 'Related Information' section and REMOVE the added `Related Information`
- Uncheck the following field: `Enable standalone Resources and support page for this Q&A.`
- [x] Select SAVE and confirm you are able to save with no issues


### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
